client/src/components/Navbar.jsx

```diff
@@ -12,24 +14,31 @@ const Navbar = (props) => {
             <BiHomeAlt/>
           </div>
         </IconContext.Provider>
-        <span className='navbar-button-text'>Portfolio</span>
+
+          <Link to="/login" className='navbar-button-text'>Portfolio</Link>
```

Comment:
Can you put the Link as a wrapper for the entire button area? currently you have to click the text to change the route. I put a suggestion revision above. The font of the "Portfolio" title looks different but the clicking action works better that way.

Example: https://stackoverflow.com/questions/42463263/wrapping-a-react-router-link-in-an-html-button

Reply:
Sure thing, I will take care of that this afternoon.  I thought I had done that, but I guess I just imagined that I did it.....